src/workloads/query/plan_cache/EmptyGroup.yml

```diff
@@ -0,0 +1,253 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Keywords:
+  - query
+  - plan_cache
+  - group
+Description: |
+  This test was created to compare using the Classic vs SBE plan caches, for an SBE query,
+  by testing a worst case.
+
+  The test uses a $group query to ensure the query is SBE-eligible, but uses an empty collection
+  to minimize the query execution time--to make the query planning time a higher proportion of
+  the overall request latency.
+
+  The sources of overhead are:
+
+    - The Classic plan cache does not store an SBE plan, so we have to run stage-builders
```

Comment:
[optional uber uber nit] the last paragraph looks like it's missing a "-" even though it's part of the second bullet. Maybe put on line 15 "the two sources of overhead are: ..."? Up to you.